### ISSUE TITLE
Ability to show Discrepancies / Unknown / Unaccounted entities

### ISSUE THREAD

**Issue by Madelena** [MindFreeze/ha-sankey-chart#2]
When there are two sets of values in two columns sourced from separate sensors, it is inevitable that there will be discrepancies.

For example, if I have my own calculations of the estimated total energy use and also have the smart meter reading from my electric company, the two values will likely not add up. It will be helpful sometimes to have the ability to show that discrepancy as a branch without the need to create another template sensor for that discrepancy.

**Describe the solution you'd like**
It would be probably be something along the lines of...

```
sections:
  - entities:
      - entity_id: sensor.total_energy_use_on_fossil_fuel
        name: Fossil Fuel
        children:
          - sensor.total_energy_use
      - entity_id: sensor.total_energy_use_on_renewables
        name: Renewables
        children:
          - sensor.total_energy_use
  - entities:
      - entity_id: sensor.total_energy_use
        name: Est. Total
        children:
          - sensor.a_plug_smartenergy_metering
          - sensor.d_plug_smartenergy_metering
      show_discrepancies: true
      discrepancies_name: Unknown Appliances
  - entities:
      - entity_id: sensor.a_plug_smartenergy_metering
      - entity_id: sensor.d_plug_smartenergy_metering
```

**Describe alternatives you've considered**
The alternative is to create template sensors of all the discrepancy values, which is a workable solution, but sometimes feel redundant. Besides, it wouldn't work if the discrepancy is sometimes a negative number.

Thank you! The chart looks awesome😁

**Comment by MindFreeze** [comment]
I have actually been thinking about this since the beginning but it is not a priority. 
It would be useful to show an 'Other' branch even if it not a discrepancy but just consumption/generation which is not monitored with a sensor.
I am still not 100% confident in the size calculations in order to do this but once these are stable it would be a cool feature indeed.

**Comment by MindFreeze** [comment]
added in v0.6.0

**Comment by tomlut** [comment]
I don't seem to be able to get this working correctly:
```
type: custom:sankey-chart
show_names: true
height: 300
round: 2
wide: true
sections:
  - entities:
      - entity_id: sensor.energy_from_grid_daily_peak
        name: Grid Peak Energy Used
        color: '#488fc2'
        children:
          - sensor.energy_imported_daily_total
      - entity_id: sensor.energy_from_grid_daily_offpeak
        name: Grid Offpeak Energy Used
        color: '#2f5f82'
        children:
          - sensor.energy_imported_daily_total
      - entity_id: sensor.energy_inverter_daily
        name: Solar Energy Produced
        color: '#ff9800'
        children:
          - sensor.energy_to_grid_daily
          - sensor.self_solar_consumption_daily
  - entities:
      - entity_id: sensor.energy_imported_daily_total
        name: Daily Imported Total
        children:
          - sensor.energy_consumed_daily_total
      - entity_id: sensor.self_solar_consumption_daily
        name: Self Consumed Solar
        color: '#ff9800'
        children:
          - sensor.energy_consumed_daily_total
      - entity_id: sensor.energy_to_grid_daily
        name: Solar Energy Exported
        color: '#8353d1'
  - entities:
      - entity_id: sensor.energy_consumed_daily_total
        name: Total Energy Used
        color: '#488FC2'
        remaining: Unmonitored
        children:
          - sensor.bar_fridge_daily_energy
          - sensor.cinema_av_daily_energy
          - sensor.comms_rack_daily_energy
          - sensor.dishwasher_daily_energy
          - sensor.electronics_bench_daily_energy
          - sensor.fridge_daily_energy
          - sensor.energy_downstairs_heat_pump_daily
          - sensor.energy_hot_water_daily_peak
          - sensor.energy_hot_water_daily_offpeak
          - sensor.led_table_daily_energy
          - sensor.lounge_av_daily_energy
          - sensor.lounge_dehumidifier_daily_energy
          - sensor.electric_blanket_daily_energy
          - sensor.mechanical_bench_daily_energy
          - sensor.pc_monitor_daily_energy
          - sensor.rack_fan_daily_energy
          - sensor.rumpus_dehumidifier_daily_energy
          - sensor.spb_electric_blanket_daily_energy
          - sensor.cinema_subwoofer_daily_energy
          - sensor.energy_upstairs_heat_pump_daily_peak
          - sensor.energy_upstairs_heat_pump_daily_offpeak
          - sensor.washing_machine_daily_energy
  - entities:
      - entity_id: sensor.bar_fridge_daily_energy
        name: Bar Fridge
        color: '#039BE5'
      - entity_id: sensor.cinema_av_daily_energy
        name: Cinema AV Gear
        color: '#039BE5'
      - entity_id: sensor.comms_rack_daily_energy
        name: Comms Rack
        color: '#039BE5'
      - entity_id: sensor.dishwasher_daily_energy
        name: Dishwasher
        color: '#039BE5'
      - entity_id: sensor.electronics_bench_daily_energy
        name: Electronics Bench
        color: '#039BE5'
      - entity_id: sensor.fridge_daily_energy
        name: Fridge
        color: '#039BE5'
      - entity_id: sensor.energy_downstairs_heat_pump_daily
        name: Downstairs Heat Pump
        color: '#039BE5'
      - entity_id: sensor.energy_hot_water_daily_peak
        name: Hot Water (Peak)
        color: '#039BE5'
      - entity_id: sensor.energy_hot_water_daily_offpeak
        name: Hot Water (Offpeak)
        color: '#039BE5'
      - entity_id: sensor.led_table_daily_energy
        name: LED Table
        color: '#039BE5'
      - entity_id: sensor.lounge_av_daily_energy
        name: Lounge AV Gear
        color: '#039BE5'
      - entity_id: sensor.lounge_dehumidifier_daily_energy
        name: Lounge Dehumidifier
        color: '#039BE5'
      - entity_id: sensor.electric_blanket_daily_energy
        name: Master Bedroom Elect Blanket
        color: '#039BE5'
      - entity_id: sensor.mechanical_bench_daily_energy
        name: Mechanical Bench
        color: '#039BE5'
      - entity_id: sensor.pc_monitor_daily_energy
        name: PC Monitor
        color: '#039BE5'
      - entity_id: sensor.rack_fan_daily_energy
        name: Rack Fan
        color: '#039BE5'
      - entity_id: sensor.rumpus_dehumidifier_daily_energy
        name: Rumpus Dehumidifier
        color: '#039BE5'
      - entity_id: sensor.spb_electric_blanket_daily_energy
        name: Spare Bedroom Elec Blanket
        color: '#039BE5'
      - entity_id: sensor.cinema_subwoofer_daily_energy
        name: Subwoofer
        color: '#039BE5'
      - entity_id: sensor.energy_upstairs_heat_pump_daily_peak
        name: Upstairs Heat Pump (Peak)
        color: '#039BE5'
      - entity_id: sensor.energy_upstairs_heat_pump_daily_offpeak
        name: Upstairs Heat Pump (Offpeak)
        color: '#039BE5'
      - entity_id: sensor.washing_machine_daily_energy
        name: Washing Machine
        color: '#039BE5'
```

![Screenshot 2022-05-23 at 12-36-16 Overview – Home Assistant](https://user-images.githubusercontent.com/10679300/169732800-199bf91e-4e71-41f5-b745-baa9e52a3ec1.png)

**Comment by MindFreeze** [comment]
Now that I think about it, this should actually be an error or working

**Comment by emufan** [comment]
Other is working here, but is displayed wrongly:

![image](https://user-images.githubusercontent.com/22775515/169831256-685ba61a-e0af-4235-9eab-2acdad267ce7.png)

Or is this the planned way?

**Comment by MindFreeze** [comment]
this doesn't look planned at all :)

**Comment by MindFreeze** [comment]
@tomlut  try v0.6.1 . it should work with the config you posted

**Comment by tomlut** [comment]
Still the same. The calculated difference is way off:

![Screenshot 2022-05-24 at 11-26-24 Overview – Home Assistant](https://user-images.githubusercontent.com/10679300/169930039-a476499a-fa99-4dab-bb38-cec137e671df.png)


I did clear my browser cache.

**Comment by MindFreeze** [comment]
Found the problem. It was the conversion from kWh to Wh. Fixed in v0.6.2

**Comment by tomlut** [comment]
Yep all good now. Thanks.
